### ISSUE TITLE
Test UnixPingUtilityTests.PacketSizeIsRespected(payloadSize: 1) failed in CI

### ISSUE THREAD

**Issue by danmosemsft** [dotnet/corefx#18125]
https://ci.dot.net/job/dotnet_corefx/job/master/job/osx10.12_release_prtest/1481/testReport/junit/System.Net.NetworkInformation.Tests/UnixPingUtilityTests/PacketSizeIsRespected_payloadSize__1_/
```
MESSAGE:
System.ArgumentOutOfRangeException : Index was out of range. Must be non-negative and less than the size of the collection.\nParameter name: startIndex
+++++++++++++++++++
STACK TRACE:
at System.String.LastIndexOf(String value, Int32 startIndex, Int32 count, StringComparison comparisonType) at System.Net.NetworkInformation.Tests.UnixPingUtilityTests.ParseReturnedPacketSize(String pingOutput) at System.Net.NetworkInformation.Tests.UnixPingUtilityTests.<PacketSizeIsRespected>d__1.MoveNext() --- End of stack trace from previous location where exception was thrown --- at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) --- End of stack trace from previous location where exception was thrown --- at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) --- End of stack trace from previous location where exception was thrown --- at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
```

**Comment by danmosemsft** [comment]
`            int indexOfBytesFrom = pingOutput.IndexOf("bytes from");` did not find hte string.

**Comment by danmosemsft** [comment]
The product code tries to do similar parsing, so any surprising input looks like it oculd have thes same result there.
```c#
   86:             int timeIndex = pingOutput.IndexOf("time=", StringComparison.Ordinal);
   88:             int msIndex = pingOutput.IndexOf("ms", afterTime);
```

**Comment by jkotas** [comment]
Failed in https://github.com/dotnet/corefx/pull/18167

**Comment by danmosemsft** [comment]
Failed again on OSX
https://ci.dot.net/job/dotnet_corefx/job/master/job/osx10.12_release_prtest/1579/consoleFull#76959989679fe3b83-f408-404c-b9e7-9207d232e5fc

**Comment by ericstj** [comment]
Failed again for me in https://github.com/dotnet/corefx/pull/18246: https://ci.dot.net/job/dotnet_corefx/job/master/job/osx10.12_release_prtest/1675/testReport/junit/System.Net.NetworkInformation.Tests/UnixPingUtilityTests/PacketSizeIsRespected_payloadSize__50_/

**Comment by JonHanna** [comment]
Again: https://ci.dot.net/job/dotnet_corefx/job/master/job/osx10.12_debug_prtest/1717/

**Comment by Priya91** [comment]
From the exception, it looks like the ping output, didn't have the expected output. This is unexpected, will have to login to the CI machine to investigate.

**Comment by stephentoub** [comment]
Fixed by #18404
